homeassistant/components/wiffi/__init__.py

```diff
@@ -0,0 +1,200 @@
+"""Component for wiffi support."""
+import asyncio
+from datetime import timedelta
+import errno
+import logging
+
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PORT
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.event import async_track_time_interval
+
+from .binary_sensor import BoolEntity
+from .const import DOMAIN
+from .sensor import NumberEntity, StringEntity
+from .wiffi import WiffiTcpServer
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_SERVERS = "servers"
+
+SERVER_CONFIG = vol.Schema({vol.Required(CONF_PORT): cv.port}, extra=vol.ALLOW_EXTRA)
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration should only use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
Removed configuration.yaml code.